src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -472,18 +472,22 @@ private void SendSnapshot(Stream stream)
             using (context.OpenReadTransaction())
             {
                 var earliestIndexEntry = _engine.GetFirstEntryIndex(context);
-                if (_followerMatchIndex >= earliestIndexEntry)
+               
+                if (_followerMatchIndex >= earliestIndexEntry ||
+                    _followerMatchIndex == earliestIndexEntry - 1) // In case the first entry is the next entry to send
```

Comment:
`_followerMatchIndex >= earliestIndexEntry-1` ?

Reply:
This was only to make the code more clear and to add the comment in the same line.

Reply:
ok